### PR TITLE
Translate comment in English

### PR DESCRIPTION
**Pull request by florianpdf**
English translation of comment

### CODE REVIEW

File: Pond.java [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
     public static void main(String[] args) {
 
-        // la référence riri reçoit une nouvelle instance de la classe Duck
+        // the riri reference receives a new instance of the Duck class
```
Comment (bastienapp) [inline Pond.java:5]: riri's reference receives a new instance of the Duck class

File: Pond.java [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
         Duck riri = new Duck("Riri", 8); 
-        // utilisation de getters      
+        // use of getters
```
Comment (bastienapp) [inline Pond.java:7]: getters usage

File: Pond.java [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
-        // la référence riri reçoit une nouvelle instance de la classe Duck
+        // the riri reference receives a new instance of the Duck class
         Duck riri = new Duck("Riri", 8); 
-        // utilisation de getters      
+        // use of getters        
         System.out.println("1. Duck name is : " + riri.getName());
         System.out.println("2. He was " + String.valueOf(riri.getAge()));
         
-        // utilisation d'un setter
+        // use of setter
```
Comment (bastienapp) [inline Pond.java:11]: setter usage

File: Pond.java [hunk 4]
```diff
@@ -17,19 +17,19 @@ public static void main(String[] args) {
...
+        // static method
         System.out.println("7. What the duck says : " + Duck.quack());
 
-        // deux références vers un même objet
+        // two references to the same object
         Duck mysterMask = new Duck("Myster Mask");
         Duck albertColvert = mysterMask;
 
-        // les deux pointent vers le même espace mémoire :
+        // both point to the same memory space:
```
Comment (bastienapp) [inline Pond.java:31]: both aim to the same memory space

**Review by bastienapp — CHANGES_REQUESTED** [review]

**Comment by bhubr** [comment]
Fixed by #3

**Pull request closed without merging**